### PR TITLE
[REF] mail: `MessageActionList`, review and simplify scss

### PR DESCRIPTION
**Pull request by Brieuc-brd**
Part of the overall v16 SCSS optimization/restyle, task-2704984

task-2826366

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by Brieuc-brd** [comment]
Hello @seb-odoo @alexkuhn 👋
This PR is ready for review 👍

cc. @stefanorigano

**Review by alexkuhn — APPROVED** [review]

**Comment by alexkuhn** [comment]
@robodoo r+

File: addons/mail/static/src/components/message_action_list/message_action_list.scss [hunk 1]
```diff
@@ -1,25 +1,11 @@
-.o_MessageActionList {
-    background-color: $white;
-    border: $border-width solid $border-color;
-    border-radius: $o-mail-rounded-rectangle-border-radius-sm;
-
-    &:hover {
-        box-shadow: 0 4px .5rem -.5rem black;
-    }
+.o_MessageActionList:hover {
+    box-shadow: 0 4px .5rem -.5rem $black;
```
[the remark this answers is not in the document]
Comment (Brieuc-brd) [reply addons/mail/static/src/components/message_action_list/message_action_list.scss:2]: I don't see any class or variable to reproduce this 🤔

File: addons/mail/static/src/components/message_action_list/message_action_list.scss [hunk 2]
```diff
@@ -1,25 +1,11 @@
...
 }
 
-.o_MessageActionList_action {
-    cursor: pointer;
-
-    &:hover {
-        background-color: mix($border-color, $white);
-    }
+.o_MessageActionList_action:hover {
+    background-color: mix($border-color, $white);
```
[the remark this answers is not in the document]
Comment (Brieuc-brd) [reply addons/mail/static/src/components/message_action_list/message_action_list.scss:6]: Here we're targeting a hover state, I don't think it's possible to use a class.

[the remark this answers is not in the document]
Comment (stefanorigano) [reply addons/mail/static/src/components/message_action_list/message_action_list.scss:6]: These elements are buttons, the right HTML tag should be `<button>`.
If you add to a `<button>`  the `btn btn-light` classes you should achieve the desired design, including the hover (and focus and active) variations.
@alexkuhn ?

File: addons/mail/static/src/components/message_action_list/message_action_list.scss (hunk 1 above)
[the remark this answers is not in the document]
Comment (stefanorigano) [reply addons/mail/static/src/components/message_action_list/message_action_list.scss:2]: 🤔 `$box-shadow/-sm/-lg`... 

Btw, not a big fan of these custom interactions behaviours... it's not about the design, but consistency.
Also, I'm not totally sure we need a shadow here... we'll see after the SCSS ref.

**Pull request closed without merging**